website-gatsby/templates/core/example-dof.jsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import AnimationLoopRunner from '../../src/components/animation-loop-runner';
 import AnimationLoop from '../../../examples/core/dof/app';
+import {setPathPrefix} from '@luma.gl/core'
 
 export default class Example extends React.Component {
+  componentDidMount() {
+    const RAW_GITHUB = 'https://raw.githubusercontent.com/uber/luma.gl/master';
```

Comment:
If we are going to be sprinkling this around in all the examples, is there a central place we can define this const?

Reply:
Yes but as discussed the gain is minimal at this point. Let's get it to work first.